testcases/OpTestInbandUsbInterface.py

```diff
@@ -152,13 +82,25 @@ def run_ipmi_cmd_on_host(self, i_cmd):
             raise OpTestError(l_msg)
```

Comment:
Inband IPMI[USB]: Channel tests
ipmitool -I usb channel authcap 1 4; echo $?
ipmitool -I usb channel authcap 1 4; echo $?
ipmitool -I usb channel authcap 1 4; echo $?
Error in USB session setup 

Unable to setup interface usb
Error loading interface usb
1
[PEXPECT]# ERROR

======================================================================
ERROR: test_channel (testcases.OpTestInbandUsbInterface.BasicInbandUSB)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestInbandUsbInterface.py", line 93, in test_channel
    self.run_ipmi_cmd_on_host(BMC_CONST.IPMITOOL_USB + BMC_CONST.IPMI_CHANNEL_AUTHCAP)
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestInbandUsbInterface.py", line 81, in run_ipmi_cmd_on_host
    l_msg = "IPMI: command failed %c" % l_cmd
TypeError: %c requires int or char

----------------------------------------------------------------------
Ran 1 test in 19.570s

FAILED (errors=1)


Here also can you replace with this below line.
        self.assertFalse(int(l_res[-1]), "IPMI: command failed %s" % l_cmd)